src/js/index.js

```diff
@@ -0,0 +1,194 @@
+import '../style.css';
+import counter from './counter.js';
+
+const card = document.querySelector('.cards');
+const body = document.querySelector('body');
+const modal = document.getElementById('myModal');
+const modalContent = document.querySelector('.content');
+const spaceShip = document.querySelector('.space');
+const baseUrl = 'https://api.tvmaze.com/search/shows?q=a';
+const baseApiUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/6AClVl2oXlI9tDJKRbp5/comments';
+
+const reachData = async () => {
+  const getData = await fetch(baseUrl);
+  const dataContent = await getData.json();
+  return dataContent;
+};
+reachData().then((data) => {
+  spaceShip.innerText = `(${counter(data)})`;
+});
+
+reachData().then((data) => data.forEach(
+  (e) => {
+    card.innerHTML += `
+           <div class="card">
+           <img src="${e.show.image.medium}" alt="image">
+           <div class="name-icon">
+               <P class="name">${e.show.name}</P>
+               <div class="onlike">
+               <i id="${e.show.id}" class="far fa-heart likes"></i>
+               <p><span id="${e.show.id}" class="number">0</span> likes</p>
+               </div>
+           </div>
+           <button class="comments" id="${e.show.id}">Comments</button>
+           </div>
+            `;
+  },
+));
+
+const getComment = async (id) => {
+  const getData = await fetch(`${baseApiUrl}?item_id=${id}`);
+  const dataContent = await getData.json();
+  return dataContent;
+};
```

Comment:
- [ ] Let's check this function to avoid getting a `400` bad request status code when opening an item that doesn't have any comments.

![Screenshot from 2021-11-26 18-17-27](https://user-images.githubusercontent.com/40503704/143614582-f6868015-470d-4c3a-844b-c00d12f22ba7.png)

Reply:
Hello thank you for the review but you can't block this error with webpack.After this feedback. I checked another project every project has the same error. Also, I checked projects all of them approved 

I tried try-catch after I tried to catch with the promise method